slider/beatmap.py

```diff
@@ -297,6 +306,21 @@ def _time_modify(self, coefficient):
 
         return type(self)(**kwargs)
 
+    def _get_combo_bits(self):
+        # bit numbers below are zero indexed.
+
+        # type code (bits number 0, 1, 3, and 7)
+        bits = self.type_code
+        # new combo (bit 2)
+        bits |= self.new_combo << 2
+        # first (MSB) of combo_skip (bit 4)
```

Comment:
Why did you make the code like this. Its so hard to understand. Also the comment is wrong

Reply:
I'm not particularly happy with it either, do you have a better suggestion? And can you elaborate on how the comment is wrong?

Reply:
it says second bit of combo skip twice.
Please just use my implementation of this method. It was good as it was.

Reply:
on second look this does seem to be more confusing than it's worth (too explicit for its own good). I've changed to your implementation